mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java

```diff
@@ -96,6 +100,74 @@ public void event(Event event) throws Exception {
             assertThat(rateExceeded).isFalse();
         }
 
+        @Test
+        default void groupNotificationShouldDeliverASingleEventToAllListenersAtTheSameTime() {
+            CountDownLatch countDownLatch = new CountDownLatch(1);
```

Comment:
Shouldn't the barrier of this latch be higher so that we assert that the 03 listeners were executed simultaneously?

Reply:
If you look carefully, It's never unlocked (until the test cleanup) so no matter the number I put here, it will block the listeners forever.